_data/projects/Python-ChatBot.yml

```diff
@@ -0,0 +1,11 @@
+name: Python-ChatBot
+desc: This is a program which perform actions on user text commands such as opening apps,performing google search and playing games.(A mini assitant)
```

Comment:
@Aswin-Poovadan I think there's a Typo at the end for `assistant`

Reply:
Solved,Thanks